packages/trader/src/Stores/Modules/Contract/Helpers/logic.js

```diff
@@ -51,6 +51,9 @@ export const isContractElapsed = (contract_info, tick) => {
     return false;
 };
 
+export const isEndedBeforeCancellationExpired = contract_info =>
+    getEndTime(contract_info) < contract_info.cancellation.date_expiry;
```

Comment:
I think `cancellation` will be undefined for non `Deal cancellation` Multiplier contract

Reply:
I agree, thank you. Fixed it.